gapic/api/artman_bigtable.yaml

```diff
@@ -6,46 +6,65 @@ common:
   gapic_api_yaml:
-    - ${REPOROOT}/googleapis/google/bigtable/v2/bigtable_gapic.yaml
+    - ${GOOGLEAPIS}/google/bigtable/v2/bigtable_gapic.yaml
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-cloud-bigtable
 go:
   gapic_code_dir: ${REPOROOT}/gapi-cloud-bigtable-go
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
-    gapic_subpath: bigtable/apiv2
+  git_repos:
+    go:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+      gapic_subpath: bigtable/apiv2
+    staging:
+      gapic_subpath: google-cloud-bigtable-v2/cloud.google.com/go/bigtable/apiv2
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-bigtable
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-bigtable-v2
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-bigtable
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/bigtable/spi/v2
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/bigtable/spi/v2
+      location: git@github.com:GoogleCloudPlatform/google-cloud-java.git
+    staging:
+      gapic_subpath: generated/java/google-cloud-bigtable-v2
 nodejs:
   gapic_code_dir: ${REPOROOT}/artman/output/js/bigtable-v2
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-node.git
-    gapic_component: src/v2
-    gapic_subpath: packages/bigtable/src/v2
+  git_repos:
+    nodejs:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-node.git
+      gapic_component: src/v2
+      gapic_subpath: packages/bigtable/src/v2
+    staging:
+      gapic_subpath: generated/nodejs/google-cloud-node/packages/bigtable
   skip_packman: True
 php:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-php-cloud-bigtable
-  git_repo:
-    gapic_subpath: generated/php/google-cloud-bigtable-v2
+  git_repos:
+    staging:
+      gapic_subpath: generated/php/google-cloud-bigtable-v2
   enable_batch_generation: True
 python:
   gapic_code_dir: ${REPOROOT}/artman/output/gapic-google-cloud-bigtable-v2
-  git_repo:
-    gapic_subpath: generated/python/gapic-google-cloud-bigtable-v2
-    grpc_subpath: generated/python/proto-google-cloud-bigtable-v2
+  git_repos:
+    staging:
+      gapic_subpath: generated/python/gapic-google-cloud-bigtable-v2
+      grpc_subpath: generated/python/proto-google-cloud-bigtable-v2
 ruby:
-  gapic_code_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-bigtable
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-ruby.git
-    gapic_component: lib/google/cloud/bigtable/v2
-    gapic_subpath: google-cloud-bigtable/lib/google/cloud/bigtable/v2
+  gapic_code_dir: ${REPOROOT}/artman/output/ruby/google-cloud-bigtable
+  git_repos:
+    ruby:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-ruby.git
+      gapic_component: lib/google/cloud/bigtable/v2
+      gapic_subpath: google-cloud-bigtable/lib/google/cloud/bigtable/v2
```

Comment:
How this is used conceptually is "cp `{generated code}/{component} -rf {destination repo}/{subpath}".

Even though a lot of it looks duplicative, there is an important distinction. In Node and Ruby, the GAPIC gets "tucked" inside of an existing manual library. Since this process involves removing everything at `subpath` before copying the new components, this staging process would effectively delete the manual layer completely.

Additionally, in Node and Ruby, only a subset of the generated code is used in these cases (the package metadata and outer `index` files are thrown away). However, the entire generated code is used in GAPIC only (so the settings would be distinct for a GAPIC only library).

Basically: this is not _actually_ redundant; there is a specific semantic meaning, which is to copy only this slice of the GAPIC into this particular slice of the target repo.